backend/src/main/java/com/crm/model/db/CustomerEntity.java

```diff
@@ -41,4 +42,16 @@
     )
     private Set<CarEntity> cars = new HashSet<>();
 
+    @Override
+    public boolean equals(Object o) {
```

Comment:
remove if we are using lombok annotation for equals and hashCode

Reply:
done